client/src/components/modal-forms/EditParticipantForm.js

```diff
@@ -25,6 +25,7 @@ export const EditParticipantForm = ({ initialValues, validationSchema, onSubmit,
               label='* Email Address'
               type='email'
             />
+            <Field name='postalCode' component={RenderTextField} label='Postal Code' type='text' />
```

Comment:
nitpick: inconsistent formatting with surrounding fields

Reply:
I think this it the right way, prettier breaks components down once they reach a certain line length.

Reply:
it's the `prettier` way, but is it the _prettier_ way? 🤔

Reply:
Praise be to prettier, heresy begone

Reply:
your dogma will be our undoing